core/urls.py

```diff
@@ -16,3 +15,10 @@
     path("", include(("home.urls", "users"), namespace="home")),
     path("carting/", include(("carting.urls", "carting"), namespace="carting")),
 ]
+
+if settings.DEBUG:
```

Comment:
```suggestion
if "django_browser_reload" in settings.INSTALLED_APPS:
```

Suggestion : on pourrait se baser sur les apps installées plutôt que sur `DEBUG`, si on change la conditionnelle qui ajoute `django_browser_reload` dans `settings.py` à l'avenir on n'aura pas à répercuter le changement dans d'autres fichiers